### PR TITLE
Create metal3 namespace for CAPM3-v1a3

### PR DESCRIPTION
**Pull request by kashifest**
Create metal3 namespace for CAPM3-v1a3 also.

### CODE REVIEW

**Comment by smoshiur1237** [comment]
Looks good to me.

**Review by fmuyassarov — APPROVED** [review]

File: 03_launch_mgmt_cluster.sh [hunk 1]
```diff
@@ -546,10 +546,9 @@ clone_repos
 create_clouds_yaml
 if [ "${EPHEMERAL_CLUSTER}" != "tilt" ]; then
   start_management_cluster
+  kubectl create namespace metal3
```
Comment (smoshiur1237) [inline 03_launch_mgmt_cluster.sh:549]: Does this line need to be inside the if statement for v1alpha3 ?

Comment (kashifest) [reply 03_launch_mgmt_cluster.sh:549]: its not right? its outside already

**Review by maelk — APPROVED** [review]

**Pull request merged**